src/main/java/org/broadinstitute/hellbender/utils/Nucleotide.java

```diff
@@ -3,22 +3,49 @@
 import java.util.Arrays;
 import java.util.Collections;
 import java.util.List;
-import java.util.stream.LongStream;
 
 /**
  * Represents the nucleotide alphabet with support for IUPAC ambiguity codes.
  *
  * <p>
  *    This enumeration not only contains standard (non-ambiguous) nucleotides, but also
- *    values to represent ambiguous and invalid codes.
+ *    values to represent ambiguous and an invalid nucleotide call ({@link #X} aka {@link #INVALID}.
  * </p>
  *
+ * <p>
+ *     You can query whether a value refers to a non-ambiguous nucleotide with {@link #isStandard()} or
+ *     {@link #isAmbiguous()} depending of your preference. Notice that the special value {@link #X}
```

Comment:
`X` is grouped together with the ambiguous nucleotides below in line 95.  Perhaps add a line break to avoid confusion.